Conference website

tonnymuchui opened the pull request:
- I worked on HTML, CSS, and javascript for this project
- featured speakers are dynamically reached from javascript whereby I created an array containing the data.
- the page is responsive.
- live version URL is: https://tonnymuchui.github.io/Conference/
- Video URL: https://www.loom.com/share/81b21ce942f949178731aae7a3bbff0f

arslanbisharat commented on about.html:
```diff
@@ -0,0 +1,94 @@
+            <div class="openMenu"><i class="fa fa-bars"></i></div>
+            <ul class="mainMenu">
+                <a href="#" class="nav-logo">Tonny's Farm</a>
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
```
- [ ] I can move from Home page to About page but I cannot move from About page to Home page. It won't let me click on any button. Please make sure you make it work as stated in the requirement.

arslanbisharat commented on about.html:
```diff
@@ -0,0 +1,94 @@
+        </nav>
+    </header>
+    <section class="landing">
+        <div class="about-landing-content">
```
- [ ] The page is not responsive. Please make sure your website is totally responsive.
![image](https://user-images.githubusercontent.com/58504655/190701231-f8fa5596-b907-4628-842d-cbbe130dc575.png)

arslanbisharat commented on about.html:
```diff
@@ -0,0 +1,94 @@
+            </div>
+        </div>
+    </section>
+    <div class="footer-class2">
+        <div class="prg1">
+            <h3 class="program">Partners</h3>
+            <hr class="underline">
+            <a class="bold">mozilla</a>
+            <a class="faint">Google</a>
+            <a class="bold">NAVER</a>
+            <a class="faint">daumkakao</a>
+            <a class="bold"><i class="fa fa-airbnb"></i>airbnb</a>
+        </div>
+    </div>
+    <script src="./js/script.js" async defer></script>
+</body>
+</html>
```
- [ ] There is no Menu button present on the about page. Try to add it there too.

arslanbisharat commented on css/about.css:
```diff
@@ -0,0 +1,221 @@
+}
+
+.tonnys-logo {
+  width: 50% !important;
+  height: 50% !important;
```
- [ ] Please don't use !important in CSS. important flag are applied to the same element, the declaration with a greater specificity is applied. Using ! important to override specificity is considered a bad practice and should be avoided for this purpose.

arslanbisharat reviewed (changes requested):
Hi @tonnymuchui, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tonnymuchui commented:
- fixed navigation links click on the about page
- fixed responsive of about page
- fixed the Menu button on about navigation
- removed `!important` in about CSS

franclobo commented on index.html:
```diff
@@ -0,0 +1,192 @@
+                <h3>Agri-investment</h3>
+                <p>Listen to the speaker from various countries about sharing and opening</p>
+            </div>
+        </div>
```
- [ ] **[OPTIONAL]** Kindly add a space between the cards and use the rest of the iconography to look similar to the original design. 👇🏽 

**Original design**

![image](https://user-images.githubusercontent.com/58642949/190832601-d7b61b25-45ff-45f9-9af6-e5e2900c21ed.png)

**Your design**

![image](https://user-images.githubusercontent.com/58642949/190832659-d3b0a319-d737-4e49-b88d-1ab33d34797b.png)

franclobo commented on js/script.js:
```diff
@@ -0,0 +1,200 @@
+const cards = [
+  {
+    id: 1,
+    imageM: './images/Picture3.png',
```
- [ ] **[OPTIONAL]** You have repeated the same image for all the speakers, please 🙏🏽 replace each one with the corresponding picture of the speaker.

**Original design**

![image](https://user-images.githubusercontent.com/58642949/190832992-100a42be-e32b-4aad-b53c-ac7f20cdd355.png)

**Your design**

![image](https://user-images.githubusercontent.com/58642949/190833013-abe2ff71-44ff-45b9-b3b8-31f3010fcc21.png)

franclobo commented on index.html:
```diff
@@ -0,0 +1,192 @@
+                    <h5>Speaker</h5>
+                    <p>Listen to the speaker from various countries about sharing and opening</p>
+                </div>
+            </div>
```
- [ ] **[OPTIONAL]** kindly use dynamics elements to complete the Feature Speakers section. 👀

franclobo commented on js/script.js:
```diff
@@ -0,0 +1,200 @@
+moreButton.addEventListener('click', showB);
+lessButton.addEventListener('click', hideB);
+
+const cards = [
```
- [ ] **[OPTIONAL]** Please 🙏🏽 complete your array, it should have six objects with the information of the speakers to complete the corresponding section dynamically.

franclobo commented on js/script.js:
```diff
@@ -0,0 +1,200 @@
+  {
+    id: 5,
+    title: 'Agri-investment',
+    imageM: './images/lecture.png',
```
- [ ] **[OPTIONAL]** You have repeated the same image for all the icons, please 🙏🏽 replace each one with the corresponding icon.

**Original design**

![image](https://user-images.githubusercontent.com/58642949/190833842-1551de1e-420e-43d3-a121-e6c3341a09f8.png)


**Your design**

![image](https://user-images.githubusercontent.com/58642949/190833836-2acc0e89-479f-4677-9fd6-c0ac8406c58f.png)

franclobo commented on about.html:
```diff
@@ -0,0 +1,97 @@
+            <div class="overlay">
+                <p class="overlay-text">Conference 2022</p>
+            </div>
+        </div>
```
- [ ] **[OPTIONAL]** Kindly add the corresponding text over the images. 👇🏽
- [ ] **[OPTIONAL]** Please 🙏🏽 use camelCase or sneak_case for the name of your classes we could write `imageLeft` or `image_left` instead of `imageleft` for the best practices of HTML & CSS. 

**Original design**

![image](https://user-images.githubusercontent.com/58642949/190834214-56c717f0-3f90-4065-acb8-7ee7d1fe70c8.png)

**Your design** 

![image](https://user-images.githubusercontent.com/58642949/190834264-d7a652ce-76f7-4255-8020-a0e84352f0e2.png)

franclobo commented on about.html:
```diff
@@ -0,0 +1,97 @@
+            </div>
+        </div>
+    </section>
+    <div class="footer-class2">
+        <div class="prg1">
+            <h3 class="program">Partners</h3>
+            <hr class="underline">
+            <a class="bold">mozilla</a>
+            <a class="faint">Google</a>
+            <a class="bold">NAVER</a>
+            <a class="faint">daumkakao</a>
+            <a class="bold"><i class="fa fa-airbnb"></i>airbnb</a>
+        </div>
+    </div>
```
- [ ] **[OPTIONAL]** Please 🙏🏽 use `footer` tag instead of `div` tag for the best practices of HTML & CSS.
- [ ] **[OPTIONAL]** Kindly add the corresponding footer in the about section to look similar to the original design. Be careful with the differences between the desk version and the mobile version. 👇🏽 

**Original design**

![image](https://user-images.githubusercontent.com/58642949/190834214-56c717f0-3f90-4065-acb8-7ee7d1fe70c8.png)

![image](https://user-images.githubusercontent.com/58642949/190834482-42d3a9be-b22d-42b4-b887-2e29e01b3fca.png)


**Your design** 

![image](https://user-images.githubusercontent.com/58642949/190834264-d7a652ce-76f7-4255-8020-a0e84352f0e2.png)

franclobo reviewed (changes requested):
Hi, @tonnymuchui, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

tonnymuchui commented:
- I added a space between the cards and used the rest of the iconography
- replaced each image with the corresponding picture of the speaker
- fixed Feature Speakers section dynamic on javascript.
- fixed array objects with the information of the speakers, to complete the corresponding section dynamically
- added the corresponding text over the images.
- I used the `footer` tag instead of `div`

levy002 reviewed (approved):
## Hello @tonnymuchui
### I'm Levy, your PR reviewer

 Excellent job for implementing all suggested changes :sunglasses: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

![](https://media.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

## Highlights
- Good job for using the correct git-flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 
- Good UI :heavy_check_mark: 
## Optional suggestions

- [OPTIONAL] KIndly make sure to add a footer on the about page in order to meet the provided design. :wink:  
- [OPTIONAL] Please make sure to remove the `PARTNERS` part on the about page in order to mee the design.
- [OPTIONAL] I highly recommend you stick to the provided design.

    - ### Your design
   ![screencapture-tonnymuchui-github-io-Conference-about-html-2022-09-17-15_08_40](https://user-images.githubusercontent.com/87197412/190858558-c2b6dea5-df95-4dbe-a53e-b26b51c8964b.png)


    - ### Provided design
  
  ![image](https://user-images.githubusercontent.com/87197412/190858640-48d82335-fad9-4e44-a5e5-2095aa8439d1.png)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
